bin/build

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -ex
+
+IMAGE_TAG="$1"
```

Comment:
I don't have strong feelings, but I would say that this naming is pretty inconsistent already, even within docker: I called it this because it is the argument to the `--tag` flag to `docker build`. Which is, confusingly, documented as specifying the `"Repository name (and optionally a tag) for the image"`. So even calling it `IMAGE` isn't really correct, since apparently we're supposed to be referring to it as the image repository? And this argument does specify the actual tag used (in addition to the repository). So ¯\\\_(ツ)\_/¯.